samples/matter/template/sample.yaml

```diff
@@ -20,15 +20,20 @@ tests:
     build_only: true
     extra_args: FILE_SUFFIX=release
       CONFIG_NCS_SAMPLE_MATTER_OPERATIONAL_KEYS_MIGRATION_TO_ITS=y
-      SB_CONFIG_BOOTLOADER_MCUBOOT=y SB_CONFIG_PARTITION_MANAGER=y
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpuapp
       - nrf7002dk/nrf5340/cpuapp
       - nrf54l15pdk/nrf54l15/cpuapp
+      - nrf54h20dk/nrf54h20/cpuapp
     platform_allow: nrf52840dk/nrf52840 nrf5340dk/nrf5340/cpuapp nrf7002dk/nrf5340/cpuapp
+<<<<<<< HEAD
```

Comment:
Some unresolved conflicts sneaked in.

Reply:
yeah thanks, seems that I forgot to save the file.